JitsiConference.js

```diff
@@ -1122,8 +1126,15 @@ function setupListeners(conference) {
                         && tracks[i].getStreamId() == streamId
                         && tracks[i].getTrackId() == trackId) {
                         var track = participant._tracks.splice(i, 1)[0];
-                        conference.eventEmitter.emit(
-                            JitsiConferenceEvents.TRACK_REMOVED, track);
+
+                        track = conference.rtc.removeRemoteTrack(
```

Comment:
Why do you overriding the track here? Do you expect the track variable to be changed after removeRemoteTrack?